enum.c

```diff
@@ -4179,13 +4179,137 @@ enum_uniq(VALUE obj)
+ *
+ * #include?, #member?:: Returns +true+ if self == object, +false+ otherwise.
+ * #all?::               Returns +true+ if all elements meet a specified criterion; +false+ otherwise.
+ * #any?::               Returns +true+ if any element meets a specified criterion; +false+ otherwise.
+ * #none?::              Returns +true+ if no element meets a specified criterion; +false+ otherwise.
+ * #one?::               Returns +true+ if exactly one element meets a specified criterion; +false+ otherwise.
+ * #count::              Returns the count of elements,
+ *                       based on an argument or block criterion, if given.
+ * #tally::              Returns a new \Hash containing the counts of occurrences of each element.
+ *
+ * === Methods for Fetching
+ *
+ * These methods return entries from the \Enumerable, without modifying it:
+ *
+ * <i>Leading, trailing, or all elements</i>:
+ * #entries, #to_a:: Returns all elements.
+ * #first::          Returns the first element or leading elements.
+ * #take::           Returns a specified number of leading elements.
+ * #drop::           Returns a specified number of trailing elements.
+ * #take_while::     Returns leading elements as specified by the given block.
+ * #drop_while::     Returns trailing elements as specified by the given block.
+ *
+ * <i>Minimum and maximum value elements</i>:
+ * #min::            Returns the elements whose values are smallest among the elements,
+ *                   as determined by <tt><=></tt> or a given block.
+ * #max::            Returns the elements whose values are largest among the elements,
+ *                   as determined by <tt><=></tt> or a given block.
+ * #minmax::         Returns a 2-element \Array containing the smallest and largest elements.
+ * #min_by::         Returns the smallest element, as determined by the given block.
+ * #max_by::         Returns the largest element, as determined by the given block.
+ * #minmax_by::      Returns the smallest and largest elements, as determined by the given block.
+ *
+ * <i>Groups, slices, and partitions</i>:
+ * #group_by::       Returns a \Hash that partitions the elements into groups.
+ * #partition::      Returns elements partitioned into two new Arrays, as determined by the given block.
+ * #slice_after::    Returns a new \Enumerator whose entries are a partition of +self+,
+                     based either on a given +object+ or a given block.
+ * #slice_before::   Returns a new \Enumerator whose entries are a partition of +self+,
+                     based either on a given +object+ or a given block.
+ * #slice_when::     Returns a new \Enumerator whose entries are a partition of +self+
+                     based on the given block.
+ * #chunk::          Returns elements organized into chunks as specified by the given block.
+ * #chunk_while::    Returns elements organized into chunks as specified by the given block.
+ *
+ * === Methods for Searching
```

Comment:
Perhaps "Methods for Searching and Filtering"?

Reply:
Done.